meta-iot2000-bsp/recipes-bsp/acpi-upgrades/acpi-upgrades.bb

```diff
@@ -7,14 +7,14 @@ inherit deploy
 
 DEPENDS = "acpica-native"
 
-SRC_URI = "file://iot2000"
+SRC_URI_iot2000 = "file://iot2000"
 
 do_compile() {
 	rm -fr ${WORKDIR}/acpi-upgrades
 
 	install -d ${WORKDIR}/acpi-upgrades/kernel/firmware/acpi
 
-	for table in ${WORKDIR}/${MACHINE}/*.asl; do
+	for table in ${WORKDIR}/*/*.asl; do
```

Comment:
I will pull all the tables that are in the work directory. What is copied to the work directory is defined by the SRC_URI, which I made machine specific.

Because the custom machine configuration prepends 'iot2000' to the MACHINEOVERRIDES it still uses this SRC_URI.